test/rpmtest/manifest.go

```diff
@@ -44,15 +45,28 @@ func PackagesFromRPMManifest(t *testing.T, r io.Reader) []*claircore.Package {
 			RepositoryHint: "key:" + rpm.GPG,
 			Module:         rpm.Module,
 		}
-		if s, ok := src[rpm.Source]; ok {
+
+		// Newer images produced from Konflux shove all the source information
+		// into the SourceName and omit the SourceNEVRA. Try both.
+		var source string
+		switch {
+		case rpm.SourceNEVRA != "":
+			source = rpm.SourceNEVRA
+		case rpm.SourceName != "":
+			source = rpm.SourceName
+		default:
+			continue
+		}
+
+		if s, ok := src[source]; ok {
 			p.Source = s
 		} else {
-			s := strings.TrimSuffix(rpm.Source, ".src")
+			s := strings.TrimSuffix(strings.TrimSuffix(source, ".rpm"), ".src")
 			pos := len(s)
 			for i := 0; i < 2; i++ {
 				pos = strings.LastIndexByte(s[:pos], '-')
 				if pos == -1 {
-					t.Fatalf("malformed NEVRA: %q", rpm.Source)
+					t.Fatalf("malformed NEVRA: %q for %q", source, rpm.Name)
```

Comment:
nit: may not be NEVRA

Reply:
We're expecting something like either 
```
acl-2.3.1-4.el9.src.rpm // SourceName
acl-0:2.3.1-3.el9.src      // SourceNEVRA
```
So I guess yeah, one is NEVRA and one is NVRA